SUPPORT.md

```diff
@@ -1,5 +1,12 @@
 # Ark Support
 
-Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list: 
+Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list and Social Platforms:
```

Comment:
```suggestion
Thanks for trying out Heptio Ark! We welcome all feedback. Here are the ways to get in touch with the Ark team:
```

Reply:
Then add all the items listed here: https://github.com/heptio/ark-community/blob/master/meetings/2018/design-session/0927/README.md#heptio-ark-community

No need to separate them by categories, they should be self-explanatory.